package.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>gz-cmake3</name>
+  <version>3.4.1</version>
+  <description>Gazebo CMake : CMake Modules for Gazebo Projects</description>
+
+  <maintainer email="grey@openrobotics.org">Michael X Grey</maintainer>
```

Comment:
Is this email still correct?

Reply:
Well, this reminded me to talk to Grey and we've decided to change the maintainer to Steve. Updated in [0a7a257](https://github.com/gazebosim/gz-cmake/pull/413/commits/0a7a257b43c760ad4356efa95114496e0d4179b6)